### DISCUSSION TITLE
Introduce new slash command "sfc" for showing the full cell content

### DISCUSSION THREAD

**Discussion by ewatch** [arnoudkooi/SN-Utils#496] [Ideas]
I often came into a situation, that i wanted to see the full content of cells, where ServiceNow truncates the content.
With this slash command one can expand all cell content in lists for Platform UI, Polaris and for both Layouts without Navigation. Related Lists should also work.

This would be solved by PR #467 which i closed for now to start discussing here.

Here is an example:
<img width="1511" alt="Bildschirmfoto 2024-02-08 um 13 58 45" src="https://github.com/arnoudkooi/ServiceNow-Utils/assets/2991341/efa2e5b1-b4dc-4159-bd63-7959588d7cbd">
<img width="1512" alt="Bildschirmfoto 2024-02-08 um 13 59 05" src="https://github.com/arnoudkooi/ServiceNow-Utils/assets/2991341/cb704961-7605-400e-ab71-e245083759a6">

Let me know what you think ?

Best regards,
Jan

**Comment by arnoudkooi** [comment]
Not sure...
This will be capped at 1000 chars, not always full content.
Would also need to check how it handles HTML fields, would they render?
